cachi2/core/package_managers/yarn/main.py

```diff
@@ -117,8 +117,9 @@ def _set_yarnrc_configuration(project: Project, output_dir: RootedPath) -> None:
     yarn_rc.enable_telemetry = False
     yarn_rc.ignore_path = True
     yarn_rc.unsafe_http_whitelist = []
-    yarn_rc.enable_mirror = True
+    yarn_rc.enable_mirror = False
```

Comment:
Is there any difference whether enableMirror is True or False? enableGlobalCache seems to be the controlling setting here for whether a local cache gets populated or not

Reply:
That is true, however, I simply didn't want to take any chances here. This particular setting is unarguably related to the cache behaviour, so even though its value doesn't matter I preferred to be explicit here that we don't want **any** cache mirroring.